lib/ansible/modules/network/eric_eccli/eric_eccli_command.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Ericsson AB.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: eric_eccli_command
+version_added: "2.9"
+author: Ericsson IPOS OAM team (@cheng)
+short_description: Run commands on remote devices running ERICSSON ECCLI
+description:
+  - Sends arbitrary commands to an ERICSSON eccli node and returns the results
+    read from the device. This module includes an
+    argument that will cause the module to wait for a specific condition
+    before returning or timing out if the condition is not met.
+  - This module also support running commands in configuration mode
+    in raw command style.
+extends_documentation_fragment: eric_eccli
```

Comment:
This documentation fragment is not required as it was meant to add provider option docs